drivers/hmis_csv_twenty_twenty/spec/models/prepend_organization_ids_spec.rb

```diff
@@ -0,0 +1,77 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+require 'rails_helper'
+
+RSpec.describe 'Prepend Organization IDs', type: :model do
+  describe 'without cleanup' do
+    before(:all) do
+      setup(with_cleanup: false)
+    end
+
+    after(:all) do
+      HmisCsvTwentyTwenty::Utility.clear!
+      GrdaWarehouse::Utility.clear!
+
+      FileUtils.rm_rf(@import_path)
+    end
+
+    it 'Has 1 organization' do
+      expect(GrdaWarehouse::Hud::Organization.count).to eq(1)
+    end
+
+    it 'Organization Name is Test Organization' do
+      expect(GrdaWarehouse::Hud::Organization.first.OrganizationName).to eq('Test Org')
+    end
+  end
+
+  describe 'with cleanup' do
+    before(:all) do
+      setup(with_cleanup: true)
+    end
+
+    after(:all) do
+      HmisCsvTwentyTwenty::Utility.clear!
+      GrdaWarehouse::Utility.clear!
+
+      FileUtils.rm_rf(@import_path)
+    end
+
+    it 'Has 1 organization' do
+      expect(GrdaWarehouse::Hud::Organization.count).to eq(1)
+    end
+
+    it 'Organization Name is (PROJECT) Test Organization' do
```

Comment:
Should be `ORG-ID`? Or change description to "has Organization ID prepended"?

Reply:
Thanks for catching that. I've adopted your suggestion for the test description.